src/content/components/language/utils.ts

```diff
@@ -15,16 +15,35 @@ function sortPackages() {
 
 export const wollokPackages = sortPackages()
 
+const wollokEntitiesRelevantMembers = (entity: Entity): List<Field|Method> => (match(entity)(
+  when(Singleton)(singleton => singleton.members),
+  when(Class)(klass => klass.members),
+  when(Mixin)(mixin => mixin.members),
+  when(Node)(_ => [])
+) as List<Field|Method>).filter(node => !node.isSynthetic)
+
 export const wollokHeadings = wollokPackages.flatMap(pkg => ([
   buildHeading(2)(pkg),
-  ...pkg.members.map(buildHeading(3))
+  ...pkg.members.flatMap(member => ([
+    buildHeading(3)(member),
+    ...wollokEntitiesRelevantMembers(member).map(buildHeading(4))
+  ])),
 ]))
 
-function buildHeading(depth: number): (node: Package | Entity) => { depth: number, slug: string, text: string } {
+export function identifier(node: Package | Entity | Field | Method): string {
+  return match(node)(
+    when(Package)(pkg => pkg.fullyQualifiedName),
+    when(Entity)(entity => entity.fullyQualifiedName),
+    when(Field)(field => `${field.parent.fullyQualifiedName}.${field.name}`),
+    when(Method)(method => `${method.parent.fullyQualifiedName}.${method.name}-${method.parameters.length}`),
```

Comment:
```suggestion
    when(Method)(method => `${method.parent.fullyQualifiedName}.${method.name}/${method.parameters.length}`),
```

Creo que en algún lado ya lo teníamos con barra (alla Prolog). Esto debería estar en TS.

Reply:
Sii y ya existe en wollok-ts, es `method.label`.

Inicialmente no lo habia usado por miedo a la `/` en urls, pero lo probe y va joya 💎